savings-account/src/ongoing_operation.rs

```diff
@@ -80,24 +79,26 @@ pub trait OngoingOperationModule {
         gas_left > MIN_GAS_TO_SAVE_PROGRESS + extra_reserve_needed + operation_cost
     }
 
+    #[inline]
```

Comment:
Normally no need for `#[inline]` unless you know for sure that there is a specific code size issue.

Reply:
I usually inline functions that are simply wrappers over other function calls, as the initial call only adds code size.